packages/react/src/components/CardLink/CardLink.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { CARD_LINK } from '../../internal/FeatureFlags.js';
```

Comment:
@photodow I just added `CARD_LINK` to react experimental, so we would need to kick off another preview build to see the card.

Reply:
pushed another commit, so this should be done. let me know otherwise.

Reply:
There it is:
https://deploy-preview-554--ibmdotcom-react-experimental.netlify.com/?path=/story/card--link